app/routers/user.py

```diff
@@ -92,3 +96,31 @@ def get_all_user_events(session: Session, user_id: int) -> List[Event]:
         session.query(Event).join(UserEvent)
         .filter(UserEvent.user_id == user_id).all()
     )
+
+
+@router.post("/disable")
+def disable_logged_user(
+        request: Request, session: Session = Depends(get_db)):
+    """route that sends request to disable the user.
+    after successful disable it will be directed to main page.
+    if the disable fails user will stay at settings page
+    and an error will be shown."""
+    disable_successful = disable(session, get_current_user)
+    if disable_successful:
+        # disable succeeded- the user will be directed to homepage.
+        url = router.url_path_for("home")
+        return RedirectResponse(url=url, status_code=HTTP_202_ACCEPTED)
```

Comment:
Good job looking into HTTP codes, but [202](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202) is actually less preferable here.
Please change to 200 OK.